server/app/routes/order/index.js

```diff
@@ -33,6 +33,17 @@ router.get('/', function(req, res, next) {
   .catch(next)
 })
 
+//Get all orders with filter (Created, Pending, etc.)
```

Comment:
merge conflict here. should copy over code from master into here and then re-add your changes